fixtures/american-dj/ikon-profile.json

```
@@ -0,0 +1,148 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Ikon Profile",
+  "shortName": "Ikon_profile",
+  "categories": ["Effect", "Color Changer"],
+  "meta": {
+    "authors": ["ameisso"],
+    "createDate": "2023-07-03",
+    "lastModifyDate": "2023-07-03"
+  },
+  "links": {
+    "manual": [
+      "https://d295jznhem2tn9.cloudfront.net/ItemRelatedFiles/9694/ADJ%20Ikon%20Profile%20%20-%20User%20Manual.pdf"
+    ],
+    "productPage": [
+      "https://www.adj.com/ikon-profile"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=PNvcf5S4VHg&embeds_referring_euri=https%3A%2F%2Fwww.adj.com%2F&source_ve_path=MjM4NTE&feature=emb_title"
+    ]
+  },
+  "physical": {
+    "dimensions": [403, 150, 127],
+    "weight": 2.7,
+    "power": 44,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "LED"
+    },
+    "lens": {
+      "degreesMinMax": [15, 30]
+    }
+  },
+  "availableChannels": {
+    "Shutter / Strobe": {
+      "defaultValue": 8,
+      "capabilities": [
+        {
+          "dmxRange": [0, 7],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Closed",
+          "comment": "shutter closed"
+        },
+        {
+          "dmxRange": [8, 15],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Open"
+        },
+        {
+          "dmxRange": [16, 131],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Strobe",
+          "speed": "1Hz",
+          "comment": "strobe slow fast"
+        },
+        {
+          "dmxRange": [132, 139],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Open"
+        },
+        {
+          "dmxRange": [140, 181],
+          "type": "ShutterStrobe",
+          "shutterEffect": "RampDown",
+          "comment": "fast close - slow open"
+        },
```

Pretty sure its the other way around
```suggestion
        {
          "dmxRange": [140, 181],
          "type": "ShutterStrobe",
          "shutterEffect": "RampUp"
        },
```